src/components/HeaderForm.js

```diff
@@ -500,41 +492,27 @@ class HeaderForm extends Component {
                     gamSelectOptions={this.state.gamSelectOptions}
                     bedSelect={this.state.bedSelect}
                     bedSelectOptions={this.state.bedSelectOptions}
-                    regionSelect={this.state.regionSelect}
-                    regionSelectOptions={this.state.regionSelectOptions}
-                    pathSelect={this.state.pathSelect}
-                    pathSelectOptions={this.state.pathSelectOptions}
                     handleInputChange={this.handleInputChange}
                   />
                 )}
-                {pathNamesFlag && (
-                  <PathNamesFormRow
-                    pathSelect={this.state.pathSelect}
-                    pathSelectOptions={this.state.pathSelectOptions}
-                    handleInputChange={this.handleInputChange}
-                  />
-                )}
-                {bedRegionsFlag && (
-                  <BedRegionsFormRow
-                    regionSelect={this.state.regionSelect}
-                    regionSelectOptions={this.state.regionSelectOptions}
-                    handleInputChange={this.handleInputChange}
+                {!examplesFlag && (
+                  <RegionInput
+                    pathNames={this.state.pathNames}
+                    regionInfo={this.state.regionInfo}
+                    handleRegionChange={this.handleRegionChange}
+                    region={this.state.region}
```

Comment:
The region box appears below the dropdowns to pick files in mounted files mode, but above the file upload controls in upload mode. It should probably always appear at the bottom, and so should be added last here, just before `DataPositionFormRow`.